tutorials/W2D4_ClimateResponse-Extremes&Variability/W2D4_Tutorial8.ipynb

```diff
@@ -1836,7 +1780,7 @@
    "name": "python3"
   },
   "kernelspec": {
-   "display_name": "Python [conda env:climatematch_2024] *",
+   "display_name": "Python [conda env:climatematch_2024]",
    "language": "python",
    "name": "conda-env-climatematch_2024-py"
   },
```

Comment:
looks like they were build with the conda env

Reply:
Sorry for this running gag. Resolved it ...